client/src/components/Header.scss

```diff
@@ -1,36 +1,114 @@
 .Header {
-
   .nav-wrapper {
-    background-color: rgb(95, 95, 95);
-    padding: 10px 10px 0 25px;
+    position: fixed;
+    width: 100%;
+    background-image: linear-gradient(rgb(120, 120, 120), rgb(40, 40, 40));
+    padding: 10px 10px 0 30px;
     color: white;
     height: 64px;
     box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14), 
       0 3px 1px -2px rgba(0, 0, 0, 0.12), 
       0 1px 5px 0 rgba(0, 0, 0, 0.2);
   }
 
-  .nav-logo {
-    display: inline-block;
-    width: calc(100% - 320px);
+  .header-wrapper {
+    width: 100%;
+    height: 70px;
+  }
 
+  .menu-wrap {
+    display: flex;
+    position: fixed;
+    padding: 0;
+    right: 0%;
+    top: 0%;
+    z-index: 1;
+    .toggler {
```

Comment:
As a rule of thumb - just a formatting thing - be sure to stick an empty line in between each new defintion. Makes it easier to read